src/cargo/ops/cargo_new.rs

```diff
@@ -707,6 +708,12 @@ mod tests {
             .unwrap_or(false)
         {
             paths::write(&path_of_source_file, default_file_content)?;
+
+            // Format the newly created source file with rustfmt
+            if let Err(e) = Command::new("rustfmt").arg(&path_of_source_file).output() {
+                let msg = format!("failed to format {}: {}", path_of_source_file.display(), e);
+                config.shell().warn(msg)?;
```

Comment:
I think it would be better if this used `log` instead.  I don't think we want to display any message if `rustfmt` is not installed, and most other errors are unlikely or generally not a significant consequence.

Reply:
I used `log::warn!` instead, does that make sense?